sql/mob_pools.sql

```diff
@@ -3748,7 +3748,7 @@ INSERT INTO `mob_pools` VALUES (3687,'Snow_Devil','Snow_Devil',232,0x00001601000
 INSERT INTO `mob_pools` VALUES (3688,'Snow_Gigas','Snow_Gigas',126,0x0000C50200000000000000000000000000000000,2,2,1,480,100,0,1,0,1,0,0,0,4143,131,0,0,0,1,0,126);
 INSERT INTO `mob_pools` VALUES (3689,'Snow_Lizard','Snow_Lizard',97,0x0000490100000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,239,131,0,0,0,1,0,97);
 INSERT INTO `mob_pools` VALUES (3690,'Snow_Maiden','Snow_Maiden',232,0x0000160100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,2,0,0,0,645,0,0,0,1,0,232);
-INSERT INTO `mob_pools` VALUES (3691,'Snow_Wight','Snow_Wight',227,0x00003C0200000000000000000000000000000000,1,1,0,0,100,0,0,0,0,0,0,0,290,131,0,0,0,1,0,227);
+INSERT INTO `mob_pools` VALUES (3691,'Snow_Wight_war','Snow_Wight',227,0x00003C0200000000000000000000000000000000,1,1,5,240,100,0,1,0,0,0,0,0,290,131,0,0,0,1,0,227);
```

Comment:
should its weapon be 7 for scythe instead? not sure if visual representation always equates to actual weapon though.

Reply:
it doesn't. mobs don't have equipment, its just part of their model. He'd do a damage type (slashing, piercing, blunt, h2h) in retail but wouldn't have a "scythe" or  any skill with any weapon type. The attack and accuracy get set in mob_family system and the weapon damage rating and delay are formed from the [mobs level](https://github.com/project-topaz/topaz/blob/5991d15a9a17549ca7a076b7a4a42ee3a26be262/src/map/utils/mobutils.cpp#L72)+dmgMulti and delay fields here in mob_pools, completely ignoring its weapon type field (which afaik only determines i it double swings or not, and possibly switch dmg type to give it by just assuming all swords are slashing etc.)